server/utils/user-stats.js

```diff
@@ -16,7 +16,7 @@ export function prepUniqueDays(cals, tz = 'UTC') {
 export function calcCurrentStreak(cals, tz = 'UTC') {
 
   let prev = _.last(cals);
-  if (moment().tz(tz).startOf('day').diff(prev, 'days') > daysBetween) {
+  if (moment().tz(tz).diff(prev, 'days') > daysBetween) {
```

Comment:
Removed this because startOf day automatically resets the day to 0 which can cause a negative infinity (-0)

![image](https://cloud.githubusercontent.com/assets/4457953/23284057/5e8c9cf8-f9f7-11e6-94d0-c1817b7fefd9.png)